.changeset/weak-chairs-judge.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hashicorp/react-version-select': patch
+---
+
+Adds additional test coverage. Depends on https://github.com/hashicorp/react-components/pull/369
```

Comment:
If we're just adding tests, we can omit the changeset!

Reply:
TIL! Removing...